src/templates/layouts/default.nunj

```diff
@@ -2,12 +2,19 @@
 <head>
   <title>{{ title }}</title>
   <link rel="stylesheet" href="/assets/bootstrap.min.css" />
+  <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700&amp;subset=cyrillic" rel="stylesheet">
   <link rel="stylesheet" href="/css/main.css" />
+  <script src="https://unpkg.com/feather-icons"></script>
 </head>
 <body>
+
   {% block content %}{% endblock %}
+  {% block carousel %}{% endblock %}
```

Comment:
I think carousel is not a part of a default layout.
Imagine pages that don't have carousel.

Please use carousel block inside the respective page (for example `index.nunj`), not in this global layout.